db/schema.rb

```diff
@@ -44,6 +44,8 @@
   create_table "events_users", id: false, force: :cascade do |t|
     t.integer "event_id", null: false
     t.integer "user_id", null: false
+    t.index ["event_id", "user_id"], name: "index_events_users_on_event_id_and_user_id"
```

Comment:
I was wondering: Is there an explanation for why this is in here?

Reply:
No idea! Probably generated after `rails db:migrate`. What would you prefer? Ignore it or keep it out of this PR?